src/controllers/auth.controller.js

```diff
@@ -38,8 +38,16 @@ function login(req, res, next) {
             username: user.username,
             email: user.email,
             scopes: user.scopes,
+            verifiedContactMethods: user.verifiedContactMethods,
         };
 
+        // check to see if the user needs to be verified to sign in, and if they are verified
+        if ((config.requireVerificaiton || config.requireSecureVerificaiton) &&
+            !userInfo.verifiedContactMethods.includes(userInfo.email)) {
+            const err = new APIError('User is not Verified', 'USER_IS_NOT_VERIFIED', httpStatus.NOT_FOUND, true);
```

Comment:
My Auth Service error improvements PR slightly changes the signature of the `APIError` constructor. All calls to the constructor added in this PR will need to be made to conform to it (if you want, we can merge this PR in first, then rebase the error improvements PR, and make all these changes after that).

In this case, it will be as simple as simply removing the last argument (the `true`).

Reply:
Merged #90 first, so indeed this must change.

Reply:
I made these changes in this branch. https://github.com/amida-tech/amida-auth-microservice/pull/91/commits/23e49317657f4407c3ba4520ea54d89baf436969